ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/normalization/UpdateNormalizer.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.tools.integritycheck.normalization
+
+import com.daml.ledger.participant.state.v1.RejectionReason.{Inconsistent, InvalidLedgerTime}
+import com.daml.ledger.participant.state.v1.Update
+
+trait UpdateNormalizer {
+  def normalize(update: Update): Update
+}
+
+object InvalidLedgerTimeRejectionNormalizer extends UpdateNormalizer {
```

Comment:
This one already works for the 1.12 reference export, so I'm going to move it to the integration kit repo.

Reply:
This comment seems not relevant anymore, or is it?

Reply:
It's moved, already. Resolving.